extensions/emmet/package.nls.json

```diff
@@ -52,5 +52,6 @@
 	"emmetPreferencesCssWebkitProperties": "Comma separated CSS properties that get the 'webkit' vendor prefix when used in Emmet abbreviation that starts with `-`. Set to empty string to always avoid the 'webkit' prefix.",
 	"emmetPreferencesCssMozProperties": "Comma separated CSS properties that get the 'moz' vendor prefix when used in Emmet abbreviation that starts with `-`. Set to empty string to always avoid the 'moz' prefix.",
 	"emmetPreferencesCssOProperties": "Comma separated CSS properties that get the 'o' vendor prefix when used in Emmet abbreviation that starts with `-`. Set to empty string to always avoid the 'o' prefix.",
-	"emmetPreferencesCssMsProperties": "Comma separated CSS properties that get the 'ms' vendor prefix when used in Emmet abbreviation that starts with `-`. Set to empty string to always avoid the 'ms' prefix."
+	"emmetPreferencesCssMsProperties": "Comma separated CSS properties that get the 'ms' vendor prefix when used in Emmet abbreviation that starts with `-`. Set to empty string to always avoid the 'ms' prefix.",
+	"emmetPreferencesFuzzySearchMinScore": "Number between 0 and 1 defining the minimum score a match should get in order to be shown as suggestion."
```

Comment:
Use the description from https://docs.emmet.io/customization/preferences/ for `css.fuzzySearchMinScore`

Reply:
We should probably prefix this setting with `css` as it does not apply to html abbreviations

Reply:
Makes sense